src/components/Button/Button.tsx

```diff
@@ -224,6 +225,7 @@ export const Button = ({
   size,
   tabIndex,
   tooltip,
+  type,
```

Comment:
By setting the default here, you should be able to avoid the `type || 'button'` throughout the file.
```suggestion
  type = 'button',
```